msgcntr/messageforums-app/src/webapp/jsp/discussionForum/message/dfAllMessages.jsp

```diff
@@ -127,6 +127,7 @@
 		<span class="highlight"  id="maxthreaddepth" class="skip"><h:outputText value="#{msgs.cdfm_maxthreaddepth}" /></span>
 //--%>
 	<h:form id="msgForum" rendered="#{!ForumTool.selectedTopic.topic.draft || ForumTool.selectedTopic.topic.createdBy == ForumTool.userId}">
+		<%@ include file="/jsp/discussionForum/menu/forumsMenu.jsp" %>
```

Comment:
There is another include of the same file in line 95 so your are including this file twice, I'm wondering if you noticed that.

Reply:
Also this form has a redered so the tabs won't be visible when then form is not rendered.

Reply:
Thank you! I did forget to delete the original include statement.

In regards to the rendered check: If the form is not rendered, you get an error message(msgs.cdfm_insufficient_privileges_view_topic) with the possibility to navigate to the previous page. 
If the menu bar should be  also rendered  in this case this solution is not valid and must be reworked. 
But in my opinion this is a minor inconvenience. Any ideas for a better solution?


![image](https://user-images.githubusercontent.com/54806745/89181874-d134dc80-d594-11ea-9fdd-6d99c6e6bdb0.png)